sdcm/utils/benchmarks.py

```diff
@@ -0,0 +1,240 @@
+        try:
+            clone_repo(self._remoter, "https://github.com/akopytov/sysbench.git")
+            clone_repo(self._remoter, "https://github.com/KnifeyMoloko/cassandra-fio.git")
+            self._install_ubuntu_prerequisites()
+            self._build_and_install_sysbench()
+        except Exception as exc:
+            LOGGER.warning("Installing benchmark tools failed with exception:\n%s", exc)
+        return self._is_sysbench_installed()
+
+    @property
+    def node_name(self):
+        return self._node.name
+
+    def _get_db_node_instance_type(self):
+        backend = self._node.parent_cluster.params.get("cluster_backend")
+        if backend in ("aws", "aws-siren"):
+            return self._node.parent_cluster.params.get("instance_type_db")
+        elif backend in ("gce", "gce-siren"):
+            return self._node.parent_cluster.params.get("gce_instance_type_db")
+        else:
+            LOGGER.warning("Unrecognized backend type, defaulting to 'Unknown' for"
+                           "db instance type.")
+            return None
+
+    def _install_ubuntu_prerequisites(self) -> bool:
+        # pylint: disable=broad-except
+        package_list = ["make", "automake", "libtool", "pkg-config", "libaio-dev", "fio"]
+        try:
+            LOGGER.info("Installing Ubuntu prerequisites for the node benchmarks...")
+            for pkg in package_list:
+                self._node.install_package(pkg, wait_for_package_manager=False)
+            LOGGER.info("Ubuntu prerequisites for the node benchmarks installed.")
+            return True
+        except Exception as exc:
+            LOGGER.warning("Failed to install Ubuntu prerequisites for the node benchmarking tools. "
+                           "Exception:\n%s", exc)
+            return False
+
+    def _build_and_install_sysbench(self) -> bool:
+        # pylint: disable=broad-except
+        build_and_install_script = """\
+            cd ./sysbench
+            ./autogen.sh
+            ./configure --without-mysql
+            make -j
+            make install
+        """
+        try:
+            self._remoter.sudo(shell_script_cmd(build_and_install_script))
+        except Exception as exc:
+            LOGGER.warning("Failed to install sysbench from source. Exception:\n%s", exc)
+
+    def _is_sysbench_installed(self) -> bool:
+        check_cmd = "sysbench --version"
+        LOGGER.info("Checking sysbench installation...")
+        check_result = self._remoter.run(check_cmd)
+        if check_result.stdout and not check_result.stderr:
+            LOGGER.info("Check: sysbench installation version: %s", check_result.stdout)
+            return True
+        return False
+
+    def run_sysbench(self, thread_num: int = None, test_time: int = 120):
+        thread_num = thread_num or self._get_number_of_cores_on_node()
+        run_cmd = f"sysbench cpu --threads={thread_num} --time={test_time} run"
+        sysbench_run_result = self._remoter.run(run_cmd)
```

Comment:
does it require root privileges to run?
Also it is good to add ignore_status, so if the command faile, it will not crash test

Reply:
It does not require root privileges. +1 to the `ignore_status` idea, I'll add that once I'm done with the ES doc restructuring.

Reply:
I decided to wrap the execution and getting the results in a try / except clause, so the `ignore_status` should no be relevant anymore.